protocol/message.go

```diff
@@ -124,15 +137,23 @@ type DirectoryProofs struct {
 	STR []*DirSTR
 }
 
+// An STRList response includes a list of signed tree roots
+// STR. A CONIKS auditor returns this DirectoryResponse type upon an
+// AudutingRequest.
+type STRList struct {
```

Comment:
Yup :) `STRList` isn't super meaningful, although it's an accurate description of the contents of the message. Because it's a range in the STR history, I was thinking something like `STRHistoryRange` could be a bit more meaningful?